snow/engine/snowman/bootstrap/bootstrapper_test.go

```diff
@@ -274,7 +270,7 @@ func TestBootstrapperSingleFrontier(t *testing.T) {
 	}
 
 	require.NoError(bs.startSyncing(context.Background(), acceptedIDs))
-	require.Equal(snow.NormalOp, config.Ctx.State.Get().State)
+	require.Equal(snow.Bootstrapping, config.Ctx.State.Get().State)
```

Comment:
This test was previously incorrect (and the implementation had a minor bug). Because this iteration accepted a block and `1 < Maxuint64/2`, there should be another round of bootstrapping performed.